src/test/resources/org/jenkinsci/plugins/plaincredentials/SecretBytesTest/loadUnencrypted/config.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson/>
```

Comment:
An empty root `config.xml` is no longer necessary with current test harness versions.